bundle/metadata/annotations.yaml

```diff
@@ -5,10 +5,14 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: operator-certification-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.bundle.channel.default.v1: alpha
```

Comment:
Do we need this file as well? This is a duplicate of `annotationsWA/annotations.yaml`. Can we keep one of them?

Reply:
removed it from PR.   I think it's a separate discussion if the file itself needs to be in the codebase.  @bcrochet has a thread on it.